docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-php.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Build a PHP application that uses YSQL
+headerTitle: Build a PHP application
+description: Build a small PHP application using the php-pgsql driver and using the YSQL API to connect to and interact with a Yugabyte Cloud cluster.
+menu:
+  latest:
+    parent: cloud-build-apps
+    name: PHP
+    identifier: cloud-php
+    weight: 900
+type: page
+isTocNested: true
+showAsideToc: true
+---
+
+The following tutorial shows a small [PHP application](https://github.com/yugabyte/yugabyte-simple-php-app) that connects to a YugabyteDB cluster using the [php-pgsql](../../../../reference/drivers/ysql-client-drivers/#php-pgsql) driver and performs basic SQL operations. Use the application as a template to get started with Yugabyte Cloud in PHP.
+
+## Prerequisites
+
+- [PHP runtime](https://www.php.net/manual/en/pgsql.setup.php). The sample application was created using PHP 8.1 but should work with earlier and later versions.
+
+  - Homebrew users on macOS can install using `brew install php`.
```

Comment:
This list is for the php-pgsql installation step (not for the PHP runtime). So just add another prerequisite such as 
_Install the [php-pgsql](https://www.php.net/manual/en/book.pgsql.php) driver:_

and the sublist down below will belong to this new prerequisite.